packages/x-date-pickers/src/MobileDateTimePicker/tests/MobileDateTimePicker.test.tsx

```diff
@@ -100,16 +100,14 @@ describe('<MobileDateTimePicker />', () => {
       expect(onClose.callCount).to.equal(0);
 
       // Change the year view
-      fireEvent.click(screen.getByLabelText(/switch to year view/));
-      fireEvent.click(screen.getByText('2010', { selector: 'button' }));
+      await user.click(screen.getByLabelText(/switch to year view/));
+      await user.click(screen.getByText('2010', { selector: 'button' }));
 
       expect(onChange.callCount).to.equal(1);
       expect(onChange.lastCall.args[0]).toEqualDateTime(new Date(2010, 0, 1));
 
-      clock.runToLast();
-
       // Change the date
-      fireEvent.click(screen.getByRole('gridcell', { name: '15' }));
+      await user.click(screen.getByRole('gridcell', { name: '15', hidden: false }));
```

Comment:
Why was this `hidden` needed here? 🤔 🤷

Reply:
I guess it is not needed anymore. I'll remove it and if I see problems in the vitest side I can add it again.